evaluator/evaluator/tasks/predict_task.py

```diff
@@ -88,42 +88,48 @@ def do_predict(
     headers = {"Authorization": f"Bearer {token}"}
     predictions: List[Prediction] = []
     start_time = datetime.datetime.now()
-    num_samples = 8  # len(dataset)
-    for i in range(num_samples):
-        logger.debug(f"Predicting sample {i + 1}/{num_samples}")
-        reference_data = dataset[i]
-
-        if dataset_metadata.skill_type == "extractive-qa":
-            query_request = {
-                "query": reference_data["question"],
-                "skill_args": {"context": reference_data["context"]},
-                "num_results": 1,
-            }
-        elif dataset_metadata.skill_type == "multiple-choice":
-            query_request = {
-                "query": reference_data["question"],
-                "skill_args": {"choices": reference_data["choices"]},
-                "num_results": 1,
-            }
-        else:
-            skill_type = dataset_metadata.skill_type
-            raise ValueError(
-                f"Predictions on '{skill_type}' datasets are currently not supported."
-            )
 
-        response = requests.post(
-            f"{base_url}/skill-manager/skill/{skill_id}/query",
-            headers=headers,
-            data=json.dumps(query_request),
+    if dataset_metadata["skill-type"] == "extractive-qa":
+        # 62c1ae1b536b1bb18ff91ce3 # squad
+        qa_type = "context"
+    elif dataset_metadata["skill-type"] == "multiple-choice":
+        # 62c1ae19536b1bb18ff91cde # commonsense_qa
+        qa_type = "choices"
+    else:
+        skill_type = dataset_metadata["skill-type"]
```

Comment:
TypeError: 'DatasetMetadata' object is not subscriptable

=> `dataset_metadata.skill_type`